front/temporal/labs/activities.ts

```diff
@@ -268,8 +269,63 @@ export async function processTranscriptActivity(
     return;
   }
 
+  // labs_transcripts_gong_full_storage FF enables storing all Gong transcripts in a single datasource view
+  let gongFullStorage = false;
```

Comment:
If I'm reading the code correctly, we won't reach this part if the user was not a participant in the call, right?

Reply:
You're right, forgot about that part. Will dig.